docs/tutorial/vald-multicluster-on-k8s.md

```diff
@@ -0,0 +1,290 @@
+  ```
+
+## Run Example Code
+
+In this chapter, you will execute insert, search, get, and delete vectors to your Vald clusters using the example code.
+
+The [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.
+
+The example code is implemented in Go and uses [vald-client-go](https://github.com/vdaas/vald-client-go), one of the official Vald client libraries, for requesting to Vald cluster.
+
+Vald provides multiple language client libraries such as Go, Java, Node.js, Python, etc.
+
+If you are interested, please refer to [SDKs](https://vald.vdaas.org/docs/user-guides/sdks).
+
+1. Port Forward(option)
+
+   If you do not use Kubernetes Ingress, port-forward is required to make requests from your local environment.
+
+   ```bash
+   kubectl port-forward svc/vald-mirror-gateway 8080:8081 -n vald-01 & \
+     kubectl port-forward svc/vald-mirror-gateway 8081:8081 -n vald-02 & \
+     kubectl port-forward svc/vald-mirror-gateway 8082:8081 -n vald-03
+   ```
+
+2. Download dataset
+
+   Move to the working directory.
+
+   ```bash
+   cd example/client/mirror
+   ```
+
+   Download [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist), which is used as a dataset for indexing and search query.
+
+   ```bash
+   wget https://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+   ```
+
+3. Run Example
+
+   We use [example/client/mirror/main.go](https://github.com/vdaas/vald/blob/feature/mirror-gateway-example/example/client/mirror/main.go) to run the example.
+
+   This example will insert and index 400 vectors into the Vald cluster from the Fashion-MNIST dataset via [gRPC](https://grpc.io/). And then, after waiting for indexing, it will request to search the nearest vector 10 times to all Vald clusters. You will get the 10 nearest neighbor vectors for each search query. And it will request to get vectors using inserted vector id.
+
+   Run example codes by executing the below command.
+
+   ```bash
+   go run main.go
+   ```
+
+   See [GetStarted](https://vald.vdaas.org/docs/tutorial/get-started/) for an explanation of the example code.
+
+## Cleanup
+
+Last, you can remove the deployed Vald cluster by executing the below command.
+
+```bash
+helm uninstall vald-cluster-01 -n vald-01 && \
+helm uninstall vald-cluster-02 -n vald-02 && \
+helm uninstall vald-cluster-03 -n vald-03
+```
+
+You can remove all ValdMirrorTarget resources (Custom Resource) with the below command.
+
+```bash
+kubectl delete --all ValdMirrorTargets -A
+```
+
+You can remove all created Namespaces with the below command.
+
+```bash
+kubectl delete namespaces vald-01 vald-02 vald-03
+```
+
+## Next Steps
+
+Congratulation! You completely entered the World of multiple Vald clusters!
+
+For more information, we recommend you check the following:
+
+- [Configuration](https://vald.vdaas.org/docs/user-guides/configuration/)
```

Comment:
It is better to show the mirror gw configuration and network policy documents.

Reply:
Thank you for your suggestion.
I added also the mirror gw configuration and network policy document.
https://github.com/vdaas/vald/pull/2105/commits/25f8622fa28a7412e5277731f2804579edfd858c